### PR TITLE
DOCSP-35193 Confluent Cloud Kafka Deployment

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
Documents the RM with Kafka Confluent Cloud deployment method.

JIRA: https://jira.mongodb.org/browse/DOCSP-35193?filter=-1

STAGING: https://preview-mongodbjocelynmendez1.gatsbyjs.io/docs-relational-migrator/DOCSP-35193/installation/kafka-deployments/install-confluent-kafka/

BUILD LOG: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65dfa9dd11db7b0427f5b4fc

### CODE REVIEW

File: source/installation/kafka-deployments/install-confluent-kafka.txt [hunk 1]
```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+You can install Relational Migrator with Kafka using 
+`Confluent Cloud <https://docs.confluent.io/cloud/current/get-started/confluent-cloud-basics.html#ccloud>`__. 
+This deployment method provides the reliability of Kafka without having you
```
Comment (jmd-mongo) [inline source/installation/kafka-deployments/install-confluent-kafka.txt]: [suggestion]

Consider using "to" here in place of "you". 

Spitballing: "This deployment method provides the reliability of Kafka without having to manage...".

File: source/installation/kafka-deployments/install-confluent-kafka.txt [hunk 2]
```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
...
+
+   .. step:: Create a Confluent Cloud environment
+
+      Go to `Confluent Cloud <https://www.confluent.io/>`__ and create a new environment. 
+
+   .. step:: Enable schema registry
+
+      Enable schema registry and select a region.
+
+      If you have more than 1000 tables being migrated, use the Advanced schema registry tier.
```
Comment (jmd-mongo) [inline source/installation/kafka-deployments/install-confluent-kafka.txt]: [suggestion]

Consider: "If you have more than 1000 tables to migrate...".

File: source/installation/kafka-deployments/install-confluent-kafka.txt [hunk 3]
```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
...
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__. 
+
+         .. step:: Configure Kafka cluster API keys
+
+            To access your Kafka cluster, you must configure your Kafka cluster API keys. 
+            Navigate to your :guilabel:`Cluster Overview` to create your Kafka cluster API 
+            keys.
+
+            Save these configurations to add to your ``user.properties`` file
```
Comment (jmd-mongo) [inline source/installation/kafka-deployments/install-confluent-kafka.txt]: [typo]

Looks like we need a full-stop here.

File: source/installation/kafka-deployments/install-confluent-kafka.txt [hunk 4]
```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
...
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__.
+
+   .. step:: Gather additional configuration information
```
Comment (jmd-mongo) [inline source/installation/kafka-deployments/install-confluent-kafka.txt]: [suggestion]

As per Vale below, consider using "more" in place of "additional" here in the interest of simplicity.

File: source/installation/kafka-deployments/install-confluent-kafka.txt [hunk 5]
```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
...
+   .. step:: Create a Confluent Cloud environment
+
+      Go to `Confluent Cloud <https://www.confluent.io/>`__ and create a new environment. 
+
+   .. step:: Enable schema registry
+
+      Enable schema registry and select a region.
+
+      If you have more than 1000 tables being migrated, use the Advanced schema registry tier.
+      Otherwise, the Essentials tier is adequate.
```
Comment (jmd-mongo) [inline source/installation/kafka-deployments/install-confluent-kafka.txt]: [suggestion]

Consider updating this to "the Essentials tier is enough" to be more consistent with terminology usage throughout the page.

File: source/installation/kafka-deployments/install-confluent-kafka.txt [hunk 6]
```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
...
+         migrator.confluent.environment.bootstrap-server: <xx.region.provider.confluent.cloud:9092>
+         migrator.confluent.environment.rest-endpoint: <https://xxx.region.provider.confluent.cloud:443>
+
+         migrator.confluent.cloud-credentials.api-key: <Cloud API key>
+         migrator.confluent.cloud-credentials.api-secret: <Cloud API Secret>
+
+         migrator.confluent.kafka-credentials.api-key: <Cluster API key>
+         migrator.confluent.kafka-credentials.api-secret: <Cluster API Secret>
+
+   .. step:: Restart Relational Migrator
```
Comment (jmd-mongo) [inline source/installation/kafka-deployments/install-confluent-kafka.txt:134]: [question]

Should there be an example here of how to restart the relational migrator? Thank you!

File: source/installation/kafka-deployments/install-confluent-kafka.txt [hunk 7]
```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
...
+
+      If you have more than 1000 tables being migrated, use the Advanced schema registry tier.
+      Otherwise, the Essentials tier is adequate. 
+
+   .. step:: Create a cluster
+
+      Navigate to your cluster page to create a new cluster.
+
+      For most use cases, the Basic and Standard tier clusters are enough. If you 
+      are moving large volumes of data, consider the Dedicated cluster tier.
```
Comment (jmd-mongo) [inline source/installation/kafka-deployments/install-confluent-kafka.txt]: [suggestion]

Consider using "migrating" here to be more consistent with the terminology throughout the page.

File: source/installation/kafka-deployments/install-confluent-kafka.txt [hunk 8]
```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
...
+
+         migrator.confluent.cloud-credentials.api-key: <Cloud API key>
+         migrator.confluent.cloud-credentials.api-secret: <Cloud API Secret>
+
+         migrator.confluent.kafka-credentials.api-key: <Cluster API key>
+         migrator.confluent.kafka-credentials.api-secret: <Cluster API Secret>
+
+   .. step:: Restart Relational Migrator 
+
+Next Steps
```
Comment (jmd-mongo) [inline source/installation/kafka-deployments/install-confluent-kafka.txt:138]: [spacing suggestion]

I recommend removing the extra line(s) between bullet points.

**Review by jmd-mongo — CHANGES_REQUESTED** [review]
Thank you for this great start, @jocelyn-mendez1! I've left a few suggestions and questions for your consideration.

Thanks,
Joe

File: source/installation/kafka-deployments/install-confluent-kafka.txt (hunk 8 above)
Comment (jocelyn-mendez1) [reply source/installation/kafka-deployments/install-confluent-kafka.txt:138]: Done.

File: source/installation/kafka-deployments/install-confluent-kafka.txt (hunk 7 above)
Comment (jocelyn-mendez1) [reply source/installation/kafka-deployments/install-confluent-kafka.txt]: Updated!

File: source/installation/kafka-deployments/install-confluent-kafka.txt (hunk 6 above)
Comment (jocelyn-mendez1) [reply source/installation/kafka-deployments/install-confluent-kafka.txt:134]: Updated, thank you!

File: source/installation/kafka-deployments/install-confluent-kafka.txt (hunk 5 above)
Comment (jocelyn-mendez1) [reply source/installation/kafka-deployments/install-confluent-kafka.txt]: Updated.

File: source/installation/kafka-deployments/install-confluent-kafka.txt (hunk 4 above)
Comment (jocelyn-mendez1) [reply source/installation/kafka-deployments/install-confluent-kafka.txt]: Updated.

File: source/installation/kafka-deployments/install-confluent-kafka.txt (hunk 3 above)
Comment (jocelyn-mendez1) [reply source/installation/kafka-deployments/install-confluent-kafka.txt]: Updated.

File: source/installation/kafka-deployments/install-confluent-kafka.txt (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/installation/kafka-deployments/install-confluent-kafka.txt]: Updated.

File: source/installation/kafka-deployments/install-confluent-kafka.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/installation/kafka-deployments/install-confluent-kafka.txt]: Updated, thank you!

**Comment by jocelyn-mendez1** [comment]
Thanks for the feedback @jmd-mongo! Pleas let me know of any other possible changes.

**Review by jmd-mongo — APPROVED** [review]
Thank you for these changes, @jocelyn-mendez1! This one LGTM.

Thanks,
Joe

**Pull request closed without merging**